birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -234,14 +242,351 @@ SERVICES = {
+        }
+    },
+    'Malleefowl': {
+        'info': {
+            'name': 'Malleefowl',
+            'synopsis': 'A Web Processing Service for Climate Data Access and Workflows.',
+            'version': "${MALLEEFOWL_VERSION}",
+            'releaseTime': get_release_time_from_repo_tag("github", "Ouranosinc/malleefowl", "${MALLEEFOWL_VERSION}"),
+            'institution': 'Ouranos',
+            'researchSubject': 'Climatology',
+            'supportEmail': '${SUPPORT_EMAIL}',
+            'category': 'Resource/Cloud Management',
+            'tags': ['Climatology']
+        },
+        'stats': {
+            'method': '.*',
+            'route': '/malleefowl/.*'
+        },
+        'redirect': {
+            'doc': 'https://malleefowl.readthedocs.io/en/latest/',
+            'releasenotes': 'https://github.com/Ouranosinc/malleefowl/blob/master/CHANGES.rst',
+            'support': 'https://github.com/Ouranosinc/malleefowl/issues',
+            'source': 'https://github.com/Ouranosinc/malleefowl',
+            'tryme': 'https://${PAVICS_FQDN_PUBLIC}/malleefowl/',
+            'licence': '',
+            'provenance': ''
+        },
+        'monitoring': {
+            "Malleefowl": {
+                'request': {
+                    'url': 'http://${PAVICS_FQDN}:8091/wps?service=WPS&version=1.0.0&request=GetCapabilities'
+                }
+            }
+        }
+    },
+    'Finch': {
+        'info': {
+            'name': 'Finch',
+            'synopsis': 'Users of climate data are interested in specific indices such as the number of freeze-thaw cycles, the number of degree-days of cooling, the duration of heatwaves, etc. This returns annual values of the most popular climate indices.',
+            'version': FINCH_VERSION,
+            'institution': 'Ouranos',
+            'releaseTime': FINCH_RELEASE_TIME,
+            'researchSubject': 'Climatology',
+            'supportEmail': '${SUPPORT_EMAIL}',
+            'category': 'Data Manipulation',
+            'tags': ['Climatology', 'Cloud']
+        },
+        'stats': {
+            'method': '.*',
+            'route': '${TWITCHER_PROTECTED_PATH}/finch.*'
+        },
+        'redirect': {
+            'doc': 'https://finch.readthedocs.io/en/latest/index.html',
+            'releasenotes': 'https://github.com/bird-house/finch/releases',
+            'support': 'https://github.com/bird-house/finch/issues',
+            'source': 'https://github.com/bird-house/finch',
+            'tryme': 'https://ouranosinc.github.io/pavics-sdi/notebooks/climate_indices.html',
+            'licence': 'https://github.com/bird-house/finch/blob/master/LICENSE.txt',
+            'provenance': 'https://ouranosinc.github.io/pavics-sdi/provenance/index.html'
+        },
+        'monitoring': {
+            'Finch': {
+                'request': {
+                    'url': 'https://${PAVICS_FQDN_PUBLIC}${TWITCHER_PROTECTED_PATH}/finch?service=WPS&version=1.0.0&request=GetCapabilities'
```

Comment:
This now changes from `http` with internal port to `https`?  This public access is already tested in `optional-components/canarie-api-full-monitoring/`

The idea is we can potentially restrict birds from public access so only the internal access is guarantied to work everytime.

So keep all internal access here and only components not behind Twitcher can directly be tested with their public access.